plugins/index.js

```diff
@@ -100,7 +106,7 @@ export function loadPlugin(manifest) {
         script.onerror = onError;
 
         document.getElementsByTagName('head')[0].appendChild(script);
-        loadedPlugins[manifest.id] = true;
+        loadedPlugins[manifest.id] = manifest;
```

Comment:
Might be out of scope for this PR, but would it make sense for this to live in the store instead of a global?

Reply:
It absolutely should, and be passed in, but requires some refactoring, which i'd say is out of scope of this PR. There may be other things that might be good candidates to move away from keeping them global, i.e. `registerPlugin` api.

Created a spike story to keep track of such items so we can split them later.
https://mattermost.atlassian.net/browse/MM-17871